src/20_Components/amp-form.html

```diff
@@ -75,7 +76,7 @@
     The `amp-form` component displays `submit-success` or `submit-error` elements based on the response and
     renders the template data inside these two elements.
   -->
-  <form method="post" action="<%host%>/components/amp-form/submit-form-input-text" action-xhr="<%host%>/components/amp-form/submit-form-input-text-xhr"  target="_top">
+  <form method="post" action-xhr="<%host%>/components/amp-form/submit-form-input-text-xhr"  target="_top">
```

Comment:
Hm - I was looking at the documentation. 

@mkhatib: POST + action - who is right? Documentation or validator?